content/administration/maintain/on_premise.rst

```diff
@@ -143,25 +132,24 @@ Duplicate a database
 ====================
 
 You can duplicate your database by accessing the database manager on your
-server (<odoo-server>/web/database/manager). In this page, you can easily
+server (<odoo-server>/web/database/manager). On this page, you can easily
 duplicate your database (among other things).
 
 .. image:: on_premise/db_manager.gif
    :align: center
 
-When you duplicate a local database, it is **strongly** advised to change
-the duplicated database's uuid (Unniversally Unique Identifier), since this
-uuid is how your database identifies itself with our servers. Having two
-databases with the same uuid could result in invoicing problems or registration
-problems down the line.
+When you duplicate a local database, it is **strongly** advised to change the duplicated
+database’s UUID (Unniversally Unique Identifier), since this UUID is how your database identifies
+itself with our servers. Having two databases with the same UUID could result in invoicing problems
+or registration problems down the line.
```

Comment:
```suggestion
```
I don't think it's necessary to say that since Odoo automatically changes the UUID of duplicated database since Odoo 9